### PR TITLE
Fix TextBuffer signals that can invalidate/modify TextIters

### PR DESCRIPTION
**Pull request by sdroege**
Fixes https://github.com/gtk-rs/gtk/issues/981

----

CC @EPashkin @GuillaumeGomez

### CODE REVIEW

File: Gir.toml [hunk 1]
```diff
@@ -1851,13 +1851,34 @@ manual_traits = ["TextBufferExtManual"]
         [[object.function.parameter]]
         name = "override_location"
         const = true
+    [[object.function]]
+    name = "register_serialize_format"
+    # actually returns an array of bytes, not a single byte
+    ignore = true
+    [[object.signal]]
+    name = "apply-tag"
+    ignore = true
```
Comment (GuillaumeGomez) [inline Gir.toml:1860]: Please add comment for each ignore (even if it's the same!).

**Comment by EPashkin** [comment]
:+1 Thanks, @sdroege

**Comment by sdroege** [comment]
> Please add missing comments. Otherwise looks good to me!

Done :)

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**